.gitignore

```diff
@@ -8,3 +8,10 @@ backend/Cargo.lock
 
 # These are backup files generated by rustfmt
 backend/**/*.rs.bk
+
+# Files, added by Idea and VSCode IDEs
+.idea
+.vscode
+
+# Files, added by MacOS
+.DS_Store
```

Comment:
As i remember, `*.min.js` and `*.min.js.map` are autogenerated files and they should be ignored

Reply:
No.
pixi.min.js is a pixi.js but without whitespace characters.
So pixi.js weights 14Mb, while pixi.min.js - 3Mb.
pixi.min.js.map - I don't know, what is it. You can try delete it.